lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -73,19 +84,51 @@ def before_scenario_starts(scenario)
       end
 
       def before_execution_starts(execution)
-        @spinner.update(execution_info(execution, 'running'))
-        @spinner.activate
+        puts(execution_info(execution, ''))
+      end
+
+      def print(string)
+        new_line_if_needed
+        @stdout.print(string)
+        @stdout.flush
+        record_last_line(string)
       end
 
-      def on_execution_update(execution, update)
```

Comment:
:D The difference is the very same as in ruby. The reason we override the methods here is to be able to capture that in the reporter and decide how to work with it. This way, we can still put
the status label at the same line, even when people use puts. The non-cli reporter would save the stings to send it somewhere else if needed.